KEEP/initramfs.sh

```diff
@@ -30,15 +30,14 @@ if [ -n "$boot" ]; then
 	mount "$boot" /boot
```

Comment:
could you put the second sentence into a second line? this looks like it spans over 80 chars.
if you dont mind. if not i'm gonna it when pulling.